packages/localize-tools/src/formatters/xliff.ts

```diff
@@ -118,7 +127,9 @@ export class XliffFormatter implements Formatter {
             !phText ||
             phText.nodeType !== doc.TEXT_NODE
           ) {
-            throw new KnownError(`Expected <ph> to have exactly one text node`);
+            throw new KnownError(
+              `Expected <${child.nodeName}> to have exactly one text node`
```

Comment:
Feel free to punt, but this seems like a good candidate for a diagnostic

Reply:
We're not in a TypeScript context here -- do you mean something that displays similar to how a TypeScript diagnostic displays with a text snippet and underline etc.?